### PR TITLE
add separate priv (oper:message) for walking over CALLERID (umode +g)

### PR DESCRIPTION
**Pull request by jesopo**
`oper:general` seems too imprecise for granting this

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ bolded warnings in the full release notes below.
 - **Breaking:** helpops now uses +h instead of +H
 - **Breaking:** sno\_whois and the spy\_ extensions have been removed
 - **Breaking:** Using /wallops now requires the oper:wallops privilege instead of oper:massnotice
+- **Breaking:** Opers wanting to override CALLERID (usermode +g) need oper:message
```
Comment (edk0) [inline NEWS.md:44]: I don't think we need a news entry since it already says all privset configs will be broken

**Review by edk0 — COMMENTED** [review]
I don't think we need a NEWS entry given it already says all privs will need to be rewritten

File: NEWS.md (hunk 1 above)
Comment (edk0) [reply NEWS.md]: ... I thought I cancelled the comment and then added this review. Oops.

**Review by edk0 — APPROVED** [review]

File: modules/um_callerid.c [hunk 2]
```diff
@@ -103,7 +103,7 @@ allow_message(struct Client *source_p, struct Client *target_p)
 		return true;
 
 	/* XXX: controversial?  allow opers to send through +g */
```
Comment (aaronmdjones) [inline modules/um_callerid.c:105]: Should this comment still be here now?

**Pull request merged**